components/ResultsTable/index.tsx

```diff
@@ -99,6 +152,7 @@ export const ResultsTable = observer(() => {
           {!root.summary.zeroEntitlements && (
             <tr className="border-t border-content font-bold ">
               <td colSpan={2}>{tsln.resultsPage.tableTotalAmount}</td>
+              <td></td>
```

Comment:
![image](https://user-images.githubusercontent.com/3630698/160485005-cbb71a98-885f-4d8c-a7ce-b997fe53521b.png)
if this line was removed, would it move the table footer into a single line? I think ideally that is how it would look